src/plattformsupport.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PLATTFORMSUPPORT_H
+#define PLATTFORMSUPPORT_H
+
+#include <QString>
+#include <QList>
+
+// Describes what Features are Available to Each plattform
+// Only VPNFeatures that have m_checkPlattformSupport set are checked here
+class PlattformSupport
+{
+public:
+   // Checks if a Feature is Supported
+   static bool supports(const QString& featureID);
```

Comment:
What about if we move this to a single feature impl?
We could have a `features` folder with a simple `FeatureX.h` file for each feature. Each feature should say:
1. I work on this platform or not
2. I could work on this platform.. but I'm disabled because I'm not ready yet.